Xamarin.Forms.Platform.WinRT/DatePickerRenderer.cs

```diff
@@ -96,7 +96,19 @@ void UpdateMaximumDate()
 		void UpdateMinimumDate()
 		{
 			DateTime mindate = Element.MinimumDate;
-			Control.MinYear = new DateTimeOffset(mindate);
+
+			try
+			{
+				Control.MinYear = new DateTimeOffset(mindate);
+			}
+			catch(ArgumentOutOfRangeException)
+			{
+				// This will be thrown if you try to set MinYear to DateTime.MinValue when the UTC offset is positive
+				// because the time is out range once it's converted. In that case, specify the Kind as UTC 
+				// so it doesn't get converted.
+				mindate = DateTime.SpecifyKind(mindate, DateTimeKind.Utc);
```

Comment:
We actually do do that in `UpdateMaximumDate()` and that was the first thing I tried. But Date just [returns another DateTime](https://msdn.microsoft.com/en-us/library/system.datetime.date(v=vs.110).aspx) with the Time set to 0:00 so in this case the crash still happens.

I think what we should do is always specify the incoming value as Utc and avoid the extra object from getting the Date. This would eliminate the `try/catch` and also prevent similar cases where the original date might get converted to the day before (or day after for MaxDate).